scripts/install_third_party.sh

```diff
@@ -140,6 +140,16 @@ if [ ! -d "$TOOLS_DIR/pycodestyle-2.3.1" ]; then
   rm pycodestyle-2.3.1.tar.gz
 fi
 
+# Install pyjsparser.
+echo Checking if pyjsparser is installed in third_party
+if [ ! -d "$TOOLS_DIR/pyjsparser-2.5.2" ]; then
+  echo Installing pyjsparser
+  # Note that the URL redirects, so we pass in -L to tell curl to follow the redirect.
+  curl -o pyjsparser-2.5.2.tar.gz -L https://pypi.python.org/packages/17/3d/59f7f0cd6a7cdab611b36d8921c9d2d6c5564033d938ec2eb934cdca0e48/pyjsparser-2.5.2.tar.gz
```

Comment:
Can we use `pip install pyjsparser` here?

Reply:
I prefer curl if possible, actually. It's more direct, pretty much every system already has it, and we've run into permissions issues with pip before.

Reply:
Understood, thanks!

Reply:
@seanlip Yeah, we really encounter lots of issues regarding pip from new contributors, that's a good point, thanks! :)